src/templates/homepage/HeroSection.jsx

```diff
@@ -178,14 +179,19 @@ const TemplateHeroSection = (
   { hero, siteName, dropdownIsActive, toggleDropdown },
   ref
 ) => {
-  const { data: loadedImageURL } = useQuery(
-    `${siteName}/${hero.background}`,
-    () => fetchImageURL(siteName, decodeURI(hero.background)),
-    {
-      refetchOnWindowFocus: false,
-      staleTime: Infinity, // Never automatically refetch image unless query is invalidated
+  const [loadedImageURL, setLoadedImageURL] = useState("")
+  const fileName = getImagePath(hero.background)
+  const { data: mediaData } = useGetMediaHook({
+    siteName,
+    mediaDirectoryName: "images",
+    fileName,
+  })
+
+  useEffect(() => {
+    if (mediaData) {
+      setLoadedImageURL(mediaData.mediaUrl)
     }
-  )
+  }, [mediaData])
```

Comment:
why can't we just rename `data` to `loadedImageURL` here?

Reply:
`data` consists of a lot of image related data, not just the url!